spec/classroom_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../classroom'
+
+describe Classroom do
```

Comment:
* As requested in the project requirement Kindly Add unit tests for all the methods in all the classes and also all classes that you created while implementing the Decorator design pattern, as some classes and methods are missing 
![image](https://user-images.githubusercontent.com/52497246/225564962-dec264f4-28e7-4c6d-972f-9aa355fd78f4.png)